packages/components/src/components/UtilityCard/UtilityCard.tsx

```diff
@@ -77,7 +77,7 @@ export const UtilityCard: React.FC<UtilityCardProps> = ({
         padding={{ _: "d4", md: "d6" }}
         w={{ _: "56px", md: "56px" }}
       >
-        <Box.img alt="" aria-hidden h="24px" src={imageSrc} w="24px" />
+        {emoji}
```

Comment:
can we add `aria-hidden` to the wrapping div? the image used to have this attribute